SwifterSwift.podspec

```diff
@@ -27,17 +27,17 @@ Pod::Spec.new do |s|
 
   # SwiftStdlib Extensions
   s.subspec 'SwiftStdlib' do |sp|
-    sp.source_files  = 'Sources/Extensions/SwiftStdlib/*.swift'
+    sp.source_files  = 'Sources/Extensions/SwiftStdlib/*.swift', 'Sources/Extensions/Shared/RandomizableInteger.swift'
   end
 
   # Foundation Extensions
   s.subspec 'Foundation' do |sp|
-    sp.source_files  = 'Sources/Extensions/Foundation/*.swift'
+    sp.source_files  = 'Sources/Extensions/Foundation/*.swift', 'Sources/Extensions/SwiftStdlib/BinaryFloatingPointExtensions.swift'
```

Comment:
Is this alright, or should anything that's shared be in the `Shared` folder?

Reply:
I don't have any time to review this PR because it's pretty large. But we shouldn't be adding extensions on the Swift Standard Libary to things like Core Graphics or UIKit